packages/web-runtime/src/components/Topbar/UserMenu.vue

```diff
@@ -8,11 +8,22 @@
       :aria-label="$gettext('User Menu')"
     >
       <avatar-image
+        v-if="userId"
         class="oc-topbar-personal-avatar oc-flex-inline oc-flex-center oc-flex-middle"
         :width="32"
         :userid="userId"
         :user-name="user.displayname"
       />
+      <oc-avatar-item
+        v-else
+        class="oc-topbar-unauthenticated-avatar oc-flex-inline oc-flex-center oc-flex-middle"
+        :name="$gettext('User Menu login')"
+        :width="32"
+        icon="user-add"
+        icon-fill-type="line"
+        icon-color="var(--oc-color-icon-text)"
```

Comment:
shouldn't this be `oc-color-swatch-brand-default` then? the `contrast` token is the contrast color for the `default` of the same swatch.

Reply:
I'll check